agentuity/server/objectstore.py

```diff
@@ -138,14 +138,6 @@ async def put(
                     span.set_attribute("contentType", params.content_type)
                 if params.content_encoding:
                     span.set_attribute("contentEncoding", params.content_encoding)
-                if params.cache_control:
```

Comment:
no no no. don't remove all the spans. just the metadataKeys span. all the rest should stay